lib/Test/Nginx/Util.pm

```diff
@@ -2380,25 +2383,7 @@ END {
                 bail_out "No pid found.";
             }
             if (is_running($pid)) {
-                if ($Verbose) {
-                    warn "sending QUIT signal to $pid";
-                }
-
-                if (kill(SIGQUIT, $pid) == 0) { # send quit signal
-                    #warn("Failed to send quit signal to the nginx process with PID $pid");
-                }
-
-                my $max_i = 15;
-                for (my $i = 1; $i <= $max_i; $i++) {
-                    last unless is_running($pid);
-
-                    sleep $TestNginxSleep;
-                    next if $i < $max_i;
-
-                    warn "WARNING: killing nginx $pid with force...";
-                    kill(SIGKILL, $pid);
-                    waitpid($pid, 0);
-                }
+                kill_nginx($pid, '');
```

Comment:
@doujiang24 Never mind. I've just created the PR #55 for this myself. Please have a look. Thanks!